plugins/postgres/dbt/include/postgres/macros/timestamps.sql

```diff
@@ -0,0 +1,17 @@
+{% macro postgres__current_timestamp() -%}
+    now()
+{%- endmacro %}
+
+{% macro postgres__snapshot_string_as_time(timestamp) -%}
+    {%- set result = "'" ~ timestamp ~ "'::timestamp without time zone" -%}
+    {{ return(result) }}
+{%- endmacro %}
+
+{% macro postgres__snapshot_get_time() -%}
+    {{ current_timestamp() }} :: TIMESTAMP without TIME ZONE
+{%- endmacro %}
```

Comment:
Nit:
Is there a reason why we don't want to preserve the casing and spacing of the original implementation?
```suggestion
{% macro postgres__snapshot_get_time() -%}
  {{ current_timestamp() }}::timestamp without time zone
{%- endmacro %}
```